packages/hawtio/src/plugins/auth/keycloak/keycloak-service.ts

```diff
@@ -117,6 +117,11 @@ class KeycloakService implements IKeycloakService {
     const initOptions: KeycloakInitOptions = {
       onLoad: 'login-required',
       pkceMethod,
+      // required for Keycloak 23
+      // see: https://github.com/keycloak/keycloak/issues/26651
+      // and we can't switch to Keycloak 25
+      // see: https://github.com/keycloak/keycloak/issues/27624
+      useNonce: false,
```

Comment:
so should we pin the version of keycloak.js to 23.x?  related: #1133

Reply:
I'm not sure what to do yet (will get back to it later), so I only left the dev notes - I hope you don't mind ;)

Reply:
@tadayosi are you ok with this explanation? :)

Reply:
It's not a change request but a pure question :-)  I wonder what we should do for #1133. Feel free to resolve it.